GUI/MainModInfo.cs

```diff
@@ -92,6 +92,7 @@ private TreeNode UpdateModDependencyGraphRecursively(TreeNode parentNode, Module
             }
 
             var node = parentNode == null ? new TreeNode(nodeText) : parentNode.Nodes.Add(nodeText);
+            node.Name = module.name;
```

Comment:
Everything looks good, but just checking this change here... Without it I presume the dependency graph could continue to show the old mod name in the case that we double-clicked on dependency mod to reach it?